products/sle/main.pm

```diff
@@ -820,7 +820,10 @@ sub load_consoletests {
     loadtest "console/ncurses";
     loadtest "console/yast2_lan" unless is_bridged_networking;
     loadtest "console/curl_https";
-    if (check_var_array('SCC_ADDONS', 'asmm')) {
+    if (check_var_array('SCC_ADDONS', 'asmm') && !sle_version_at_least('15')) {
+        loadtest "console/puppet";
+    }
+    if (check_var_array('SCC_ADDONS', 'phm') && sle_version_at_least('15')) {
```

Comment:
we already have some package hub tests and its abbr is 'phub' and please use it to keep consistent.
see: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/registration.pm#L251